static/js/NavSidebar.jsx

```diff
@@ -669,7 +670,38 @@ const StayConnected = () => { // TODO: remove? looks like we are not using this
   );
 };
 
+const Button = ({img, href, children, classes={}}) => {
+  classes = {button: 1, ...classes};
+  return <a className={classNames(classes)} href={href}>
+          {img}
+          <InterfaceText>{children}</InterfaceText>
+          </a>
+}
+const GetStartedButton = () => {
+    const href = Sefaria.interfaceLang === 'english' ? "/sheets/393695" : "/sheets/399333";
+    return <Button classes={{getStartedSheets: 1}} href={href}>Get Started</Button>
+}
+const CreateSheetsButton = () => {
+  const img = <img src="/static/icons/new-sheet-black.svg" alt="make a sheet icon" id="sheetsButton"/>;
```

Comment:
Why do we need a const? Why not jsust render if this is all hardcoded and predefined?